src/setup.py

```diff
@@ -27,19 +27,23 @@
 
 setup(
     name="covid19",
-    version="0.1",
+    version="0.2",
     author="SWIG Docs",
     description="""Individual-based model for modelling of a COVID-19 outbreak""",
     ext_modules=[covid19_module],
+    packages=[
+        'COVID19',
+        'adapter_covid19',
+    ],
     py_modules=[
-        "covid19",
-        "COVID19/parameters",
-        "COVID19/model",
-        "COVID19/simulation",
+        'covid19',
     ],
     requires=[
```

Comment:
Changing this to `install_requires` installed the dependencies, otherwise, they do not get installed with `pip` (as is in the makefile).